app/controllers/idv/link_sent_controller.rb

```diff
@@ -36,6 +36,7 @@ def update
     def extra_view_variables
       { phone: idv_session.phone_for_mobile_flow }.merge(
         phone_question_ab_test_analytics_bucket,
+        phone_with_camera: idv_session.phone_with_camera,
```

Comment:
for consistency .. .consider restructuring to? 🤔 

{ 
  phone: idv_session.phone_for_mobile_flow,
  phone_with_camera: idv_session.phone_with_camera,
 }.merge(phone_question_ab_test_analytics_bucket)

Reply:
[Done](https://github.com/18F/identity-idp/pull/9481/commits/7523b829b1c134dfd08de3c5b7dcc8dd72d51694)